### ISSUE TITLE
(0.17) Some ideas

### ISSUE THREAD

**Issue by BurtGummer** [Oarcinae/FactorioScenarioMultiplayerSpawn#70]
Hello Oarcinae,

1. can we have one file per language for the complete translations? I have see, you have create a "locale" directory, would be great to have one file to translate everything (to german).

2. if i upgrade the scenario, i download the master.zip, extract, copy all content to my scanario patch. but thats bad, i overwrite the config.lua: maybe its better to have a not existing "config.local.lua", this file is for users to change and config things and overwrite the options in config.lua. So a script can always overwrite the config.lua.

**Comment by Oarcinae** [comment]
I went ahead and made another ticket for #1. It'll take me awhile to pull out all the text strings but I'll let you know when I get time to do it. Will ask you to help me do the german translations :)

#2 should be easy as well. And it's a good idea. I've been meaning to do something like that for awhile. The result will be that any new player will always have to rename the example config file first, before starting a game, but I think this is fine.
Also, users will have to watch out for when the config file changes, as changes may break things in unexpected ways.
